language-design.md

```diff
@@ -106,7 +172,17 @@ you do so? If not, why not?
 
 **Response**
 
+From the selected quotes, the two experiences of natural language seem to be at odds with one another. The Pavlus quote claims that users preferred working with something akin to a natural language, whereas the Cook quote says that programmers found natural language to be too ambiguous. These two experiences cannot be directly compared without examining the backgrounds of the people being quoted as well as the amount of exposure that they had to the language. 
 
+The Pavlus article suggests that the users in the usability testing were novice programmers. They preferred the natural language replacements and did not mention the depth or duration of time that these users spent working with the language. The Cook quote labels users as programmers, but it does not specify the programming abilities of the users. In short, the two experiences could be from the perspectives of different audiences. If novice programmers had little exposure to language, they might be able to relate to natural language more easily. In contrast, users of AppleScript could have more programming experience and thus more confusion over the inconsistencies of natural language syntax. Alternatively, users of AppleScript may have been developing more complex programs and run into more cases of ambiguity. 
+
+Regardless, these are only two examples of natural language in a DSL. The decision to include natural language in a DSL depends on the domain and the targeted user of the language. It is generally safer to avoid the ambiguous behavior that can result from the translation of natural language into programming syntax, especially when unpredictable users are involved. There are cases where the targeted user would benefit by interacting with the system through a natural language interface, especially if that language is simple enough to minimize confusion.
+
+We both agreed that a great example of natural language implementation in a DSL is [Scratch](https://scratch.mit.edu/). This DSL is used to create animations and games and introduces users to basic programming logic through blocks of natural language. These blocks can be combined in specific configurations to create animations. The language minimizes error by limiting the ways in which the blocks can be combined. Scratch is often used as an introduction to computer programming because it largely consists of natural language describing basic concepts of program logic. By preventing excessive error, Scratch does not introduce as much ambiguity as AppleScript but solves the problem presented in the Pavlus article that:
+
```

Comment:
I agree--using a more simple natural language approach removes a lot of the confusion of natural language. I think it could be interesting to study ways to increase complexity but still reduce ambiguity.